cypress/utils/index.js

```diff
@@ -0,0 +1,79 @@
+import { waitForGQL } from "./networking";
+
+// used to test status and base status dropdown filters
+export const clickingCheckboxFetchesFilteredTasksAndUpdatesUrl = ({
```

Comment:
can we include something in this function name to indicate that it tests the rendered items? also we should remove "tasks" since we're using this in the test table as well.